CHANGES.rst

```diff
@@ -2,10 +2,10 @@
 CHANGES
 =======
 
-2.0.1 (unreleased)
-------------------
+2.1 (unreleased)
+----------------
 
-- Nothing changed yet.
+- Add support for Python 3.5 up to 3.7.
```

Comment:
This could mention, PyPy, too.

Reply:
I forgot to save the changelog before adding. Now it also includes PyPy.